cmd/db_audit_logs_exporter/db_audit_logs_exporter.go

```diff
@@ -200,8 +211,8 @@ var removeDbAuditLogsExporterCmd = &cobra.Command{
 func init() {
 	DbAuditLogsExporterCmd.AddCommand(assignDbAuditLogsExporterCmd)
 	assignDbAuditLogsExporterCmd.Flags().SortFlags = false
-	assignDbAuditLogsExporterCmd.Flags().String("integration-id", "", "[REQUIRED] The ID of the Integration")
-	assignDbAuditLogsExporterCmd.MarkFlagRequired("integration-id")
+	assignDbAuditLogsExporterCmd.Flags().String("integration-name", "", "[REQUIRED] Name of the Integration")
+	assignDbAuditLogsExporterCmd.MarkFlagRequired("integration-name")
```

Comment:
If this is released to the public, we will need to deprecate it.

Reply:
this is unreleased and was introduced as part of my previous PRs

Reply:
Cool